crates/turbopack-tests/tests/snapshot/css/css/output/crates_turbopack-tests_tests_snapshot_css_css_input_style.css

```diff
@@ -2,13 +2,16 @@
 /* import([project]/crates/turbopack-tests/tests/snapshot/css/css/input/imported.css (css)) */
 @layer layer {
   @media print {
```

Comment:
This indention on this line is a little bit unexpected now. I would expect it to be not indented. While not perfect, I can accept that.

Reply:
It is caused by the `BasicCssWriter` in `ImportAttributes::print_block`, I can make a new PR to fix it.